MANIFEST.in

```diff
@@ -1 +1,2 @@
-recursive-include wof/flask/templates *
\ No newline at end of file
+recursive-include wof/flask/templates *
+recursive-include wof/examples/flask/odm2/timeseries *
```

Comment:
>  instead of a first class module named examples that can be easily overwriting if another package does the same.

I'm not sure I follow that scenario, but that's ok.